msteams-platform/apps-in-teams-meetings/in-meeting-notification-for-meeting.md

```diff
@@ -53,6 +53,100 @@ The following table provides the user types and lists the features that each use
 | Federated or External | Available | Not available | Not available | Available |
 | Anonymous | Not available | Not available | Not available | Not available |
 
+## Targeted in-meeting notification
+
+Targeted in-meeting notification allows apps to send notifications to specific participants in a meeting. The notifications are private and are sent only to specific or targeted participants. Targeted in-meeting notification helps to enhance meeting experience and develop user engagement activities in Teams meetings.
```

Comment:
I like the definition as it is, but let's add one more details :)
"Targeted in-meeting notification allows apps to send notifications to specific participants in a meeting **on a meeting stage**"